### PR TITLE
Set Module to ES

### PR DESCRIPTION
**Pull request by leonardssh**
## Description

I'm running ES6 modules for a project and I practically couldn't find the right inversify library, and now it works great, after 2 hours of research.

Maybe others need it too. 😊

## How Has This Been Tested?

Localhost

## Types of changes

- [x] Updated docs / Refactor code / Added a tests case (non-breaking change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the changelog.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "A powerful and lightweight inversion of control container for JavaScript and Node.js apps powered by TypeScript.",
   "main": "lib/inversify.js",
   "jsnext:main": "es/inversify.js",
+  "module": "es/inversify.js",
```
Comment (notaphplover) [inline package.json:7]: did you try `es6/inversify.js`?

Comment (notaphplover) [inline package.json:7]: did you try `es6/inversify.js`?

Comment (leonardssh) [reply package.json:7]: Yeap, it didn't work.

**Review by notaphplover — APPROVED** [review]
LGTM

**Pull request merged**